src/engine/condition.ts

```diff
@@ -15,10 +15,11 @@ const createCondition = <T>(condition: EngineCondition) => (
           return (queryFeatures || []).some(
             (v) => query.value.indexOf(v) !== -1
           );
-        } else if (query.property === 'topicModel') {
+        } else if (query.property === 'topicDist') {
```

Comment:
I should have spotted this earlier, but why does edgekit need to know about `topicDist` / `topicModel` - this PR is full of direct checks on the property name.

The actual setting of this property comes from the user or in our case the wrapper.

The only thing edgekit needs to know is what the type will be (in this case a vector), which would then drive how it was compared to the audience definition.